share/spice/thumbtack/thumbtack.js

```diff
@@ -0,0 +1,47 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_thumbtack = function(api_result){
+
+        // Don't show anything if we weren't able to return 2 or more services.
+        // In future iterations, we should support a different view in the case
+        // of a single result.
+        if (!api_result || api_result.error || api_result.data.length < 2) {
+            return Spice.failed('thumbtack');
+        }
+
+        // Render the response
+        DDG.require('maps', function() {
+            Spice.add({
+                id: "thumbtack",
+
+                name: "Professionals",
+                data: api_result.data,
+                model: 'Place',
+                view: 'Places',
+                normalize: function(item) {
+
+                    // Duckduckgo currently doesn't support caching https images,
+                    // so convert them to http for now until this is resolved.
+                    var img_url = item.image;
```

Comment:
you can use `DDG.toHTTP()` here -- it will return `undefined` if the input is undef as well